src/fontra/views/fontinfo/panel-sources.js

```diff
@@ -428,13 +452,57 @@ class SourceBox extends HTMLElement {
     }
   }
 
-  toggleShowHide() {
-    const element = this.querySelector("#open-close-icon");
-    element.classList.toggle("item-closed");
+  toggleShowHide(altKey) {
+    const cardElements = !altKey
+      ? [this]
+      : document.querySelectorAll(".fontra-ui-font-info-sources-panel-source-box");
+
+    const thisIconElement = this.querySelector("#open-close-icon");
+    const isClosed = thisIconElement.classList.contains("item-closed");
+
+    for (const cardElement of cardElements) {
+      const elementIcon = cardElement.querySelector("#open-close-icon");
+      if (isClosed) {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = false;
+        cardElement.classList.remove("item-closed");
+        elementIcon.classList.remove("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.remove("min-height");
+        }
+      } else {
+        cardsInfos[cardElement.sourceIdentifier]["isClosed"] = true;
+        cardElement.classList.add("item-closed");
+        elementIcon.classList.add("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.add("min-height");
+        }
+      }
+    }
+  }
+
+  getOneliner() {
+    const onelinerElement = html.div({
+      class: "fontra-ui-font-info-sources-panel-oneliner",
+    });
+    onelinerElement.appendChild(html.span({ class: "bold" }, [this.source.name] + " "));
```

Comment:
I would like a little more space between the source name and the location string. Either add some `&nbsp;` chars, or fiddle with the CSS.

Reply:
Done.